index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=
+    , initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body style="background-color: brown;">
```

Comment:
I have noticed that you are using inline styles in html, I kindly suggest that you create a styles.css file and add the inline rule to the css file.

Reply:
- [ ] I think it could be a good idea to explain "why" you are requesting a change, this way the code owner will learn something new and it could set a friendly tone :wink: